cmd/log-validator/main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/hpcloud/tail"
+
+	"github.com/letsencrypt/boulder/cmd"
+	blog "github.com/letsencrypt/boulder/log"
+)
+
+func lineValid(text string) bool {
+	// Line format should match the following template:
+	//   timestamp hostname datacenter level binary-name[pid]: checksum msg
+	//
+	// template( name="LELogFormat" type="list" ) {
```

Comment:
Can you explicitly mention that this is an `rsyslog` template in the comment somewhere? I think its valuable context that won't be top of mind for folks that haven't looked at an rsyslog config before.